docs/superpowers/plans/2026-08-14-skill-fork-confirmation-gate.md

```
@@ -0,0 +1,632 @@
+def test_now_skips_the_confirmation_but_never_the_naming_rules() -> None:
+    text = _text()
+    assert "### Skip the confirmation with `--now`" in text
+    assert "skips the confirmation, never the naming rules" in text
+    assert "never invents a random name" in text
+    assert "skip the dry run and the question" in text
+```
+
+- [ ] **Step 2: Run test to verify it fails**
+
+Run: `uv run pytest tests/skill/test_companion_skill.py::test_now_skips_the_confirmation_but_never_the_naming_rules -v`
+Expected: FAIL — `assert '### Skip the confirmation with `--now`' in text`.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)